### PR TITLE
add(BANger): Added settings to BANger

### PR DESCRIPTION
**Pull request by GlitchChan**
Adds a simple setting to change the video to whatever

### CODE REVIEW

**Comment by Vendicated** [comment]
why the rename to tsx? it doesn't matter I'm just curious

File: src/plugins/banger.tsx [hunk 1]
```diff
@@ -16,8 +16,16 @@ export default definePlugin({
             find: "BAN_CONFIRM_TITLE.",
             replacement: {
                 match: /src:\w\(\d+\)/g,
-                replace: 'src: "https://i.imgur.com/wp5q52C.mp4"'
+                replace: "src: Settings.plugins.BANger.source"
```
Comment (Vendicated) [inline src/plugins/banger.tsx:19]: shouldn't this be 
```suggestion
                replace: "src: Vencord.Settings.plugins.BANger.source"
```
remember that not everyone has console shortcuts enabled

**Comment by GlitchChan** [comment]
> why the rename to tsx? it doesn't matter I'm just curious

Because I was messing with things and forgot to change it back lol

File: src/plugins/banger.tsx (hunk 1 above)
Comment (GlitchChan) [reply src/plugins/banger.tsx:19]: Ah I did not know that was apart of console shortcuts

Comment (Vendicated) [reply src/plugins/banger.tsx]: for future ref the only guaranteed window props are Vencord and VencordNative, anything else is console shortcuts

**Review by Vendicated — APPROVED** [review]

**Pull request merged**